src/js/controller/model.js

```diff
@@ -229,8 +229,9 @@ const Model = function() {
 
         _provider.volume(_this.get('volume'));
 
-        // Mute the video if autostarting on mobile. Otherwise, honor the model's mute value
-        _provider.mute(_this.autoStartOnMobile() || _this.get('mute'));
+        // Mute the video if autostarting on mobile, except for Android SDK. Otherwise, honor the model's mute value
+        const isAndroidSdk = _this.get('sdkplatform') === 1;
+        _provider.mute((this.autoStartOnMobile() && !isAndroidSdk) || _this.get('mute'));
```

Comment:
`this.autoStartOnMobile()` calls  `platformCanAutostart()`, which checks `isAndroidSdk`. You can make the change there and leave this condition as it was.

Reply:
Not sure if I follow here -- won't changing `platformCanAutostart()` disable autostart in the SDK for Android?

Reply:
You're right - I was suggesting conflating autostart behavior with 'muted' autostart behavior. Your logic is correct.